app/scripts/views/mixins/session-verification-poll-mixin.js

```diff
@@ -96,7 +96,7 @@ define(function (require, exports, module) {
         // if the user is switching networks.
         // Report a known error to Sentry, but not the user.
         // Details: github.com/mozilla/fxa-content-server/issues/2638.
-        this.logError(AuthErrors.toError('POLLING_FAILED'));
+        this.logError(AuthErrors.toError('POLLING_FAILED', err));
```

Comment:
The [second parameter to `toError` is expected to be a string](https://github.com/mozilla/fxa-content-server/blob/polling-failed-details/app/scripts/lib/errors.js#L115). When trying to synthesize an XHR error that would cause an `UNEXPECTED_ERROR` to be thrown, an error is displayed in the console saying the error report was rejected because of an invalid context:


![screen shot 2017-08-10 at 11 06 12](https://user-images.githubusercontent.com/848085/29165829-0afb809c-7dbc-11e7-94ce-f0d066bf50be.png)

`context` needs to be a string because it's [imported as a tag into Sentry](https://github.com/mozilla/fxa-content-server/blob/polling-failed-details/app/scripts/lib/sentry.js#L189:L194).

I used the following patch to generate an XHR error:

```diff
diff --git a/app/scripts/models/account.js b/app/scripts/models/account.js
index 54118d9d..8cbb7d37 100644
--- a/app/scripts/models/account.js
+++ b/app/scripts/models/account.js
@@ -247,6 +247,8 @@ define(function (require, exports, module) {
      * }
      */
     sessionStatus () {
+      const err = AuthErrors.normalizeXHRError({ status: 300 });
+      return p.reject(err);
       var sessionToken = this.get('sessionToken');
       if (! sessionToken) {
         return p.reject(AuthErrors.toError('INVALID_TOKEN'));
```

Perhaps see if the error has a status and do do something like:

```js
const context = err.status && `http.${err.status}`;
this.logError(AuthErrors.toError('POLLING_FAILED', context));
```